### ISSUE TITLE
No such file or directory file in Step 06

### ISSUE THREAD

**Issue by aditya109** [mmumshad/kubernetes-the-hard-way#248]
**Describe the bug**
No such file or directory file while following steps mentioned in [Generating the Data Encryption Config and Key](https://github.com/mmumshad/kubernetes-the-hard-way/blob/master/docs/06-data-encryption-keys.md)

**To Reproduce**

1. Follow the steps from 01 to 05.

2. In step 06, execute the command to move `encryption-config.yaml` encryption config file to appropriate directory.

   ```sh
   for instance in master-1 master-2; do
     ssh ${instance} sudo mv encryption-config.yaml /var/lib/kubernetes/
   done
   ```

3. We get the following error:

   ```sh
   mv: cannot move 'encryption-config.yaml' to '/var/lib/kubernetes/': Not a directory
   ```

**Expected behavior**
No such error should be displayed.

**Screenshots**
![image](https://user-images.githubusercontent.com/22005947/137769452-8f73010d-7a5a-410c-ae01-535a0b4df292.png)

**Desktop (please complete the following information):**

 - OS: [e.g. Windows, Linux]

**Comment by patsevanton** [comment]
Reproduce issue

**Comment by aditya109** [comment]
Hi @patsevanton , I have, if you would TAL my description.

**Comment by tarikly** [comment]
That procedure is present on (https://github.com/mmumshad/kubernetes-the-hard-way/blob/master/docs/08-bootstrapping-kubernetes-controllers.md#configure-the-kubernetes-api-server).

So you can just keep following the tutorial without problems.

**Comment by aditya109** [comment]
Hi @tarikly, you are correct, but the step you mentioned is supposed to create a directory in master node(s).
My fix is meant for worker nodes.

**Comment by fireflycons** [comment]
Hi @aditya109 

We have now merged a major change to bring this to v1.24.
This has been addressed as part of the upgrade. Please try it now and feel free to raise further issues.

Thanks.
